projects/igniteui-angular/src/lib/grid/column-resizing.spec.ts

```diff
@@ -492,55 +474,51 @@ describe('IgxGrid - Deferred Column Resizing', () => {
+        fixture.detectChanges();
 
-    //     expect(grid.columns[0].width).toEqual("150px");
-    //     expect(fixture.componentInstance.count).toEqual(0);
+        const dblclick = new Event('dblclick');
+        const grid = fixture.componentInstance.grid;
+        const headers: DebugElement[] = fixture.debugElement.queryAll(By.css(COLUMN_HEADER_CLASS));
 
-    //     const headerResArea = headers[0].nativeElement.children[1];
-    //     UIInteractions.simulateMouseEvent("mousedown", headerResArea, 150, 5);
-    //     tick();
-    //     fixture.detectChanges();
+        expect(grid.columns[0].width).toEqual('150px');
+        expect(fixture.componentInstance.count).toEqual(0);
 
-    //     const resizer = headers[0].nativeElement.children[1].children[0];
-    //     expect(resizer).toBeDefined();
-    //     UIInteractions.simulateMouseEvent("mousemove", resizer, 300, 5);
-    //     tick();
+        const headerResArea = headers[0].nativeElement.children[2];
+        UIInteractions.simulateMouseEvent('mousedown', headerResArea, 150, 5);
+        tick();
+        fixture.detectChanges();
 
-    //     UIInteractions.simulateMouseEvent("mouseup", resizer, 300, 5);
-    //     tick();
-    //     fixture.detectChanges();
+        const resizer = headers[0].nativeElement.children[2].children[0];
+        expect(resizer).toBeDefined();
+        UIInteractions.simulateMouseEvent('mousemove', resizer, 300, 5);
+        tick();
 
-    //     expect(grid.columns[0].width).toEqual("300px");
-    //     expect(fixture.componentInstance.count).toEqual(1);
-    //     expect(fixture.componentInstance.column).toBe(grid.columns[0]);
-    //     expect(fixture.componentInstance.prevWidth).toEqual("150");
-    //     expect(fixture.componentInstance.newWidth).toEqual("300px");
+        UIInteractions.simulateMouseEvent('mouseup', resizer, 300, 5);
+        tick();
+        fixture.detectChanges();
 
-    //     expect(grid.columns[1].width).toEqual("150px");
+        expect(grid.columns[0].width).toEqual('300px');
+        expect(fixture.componentInstance.count).toEqual(1);
+        expect(fixture.componentInstance.column).toBe(grid.columns[0]);
+        expect(fixture.componentInstance.prevWidth).toEqual('150');
+        expect(fixture.componentInstance.newWidth).toEqual('300px');
 
-    //     headers[1].componentInstance.resizeArea.nativeElement.dispatchEvent(dblclick);
-    //     tick();
-    //     fixture.detectChanges();
+        expect(grid.columns[1].width).toEqual('150px');
 
-    //     expect(grid.columns[1].width).toEqual("207px");
-    //     expect(fixture.componentInstance.count).toEqual(2);
-    //     expect(fixture.componentInstance.column).toBe(grid.columns[1]);
-    //     expect(fixture.componentInstance.prevWidth).toEqual("150");
-    //     expect(fixture.componentInstance.newWidth).toEqual("207px");
+        headers[1].componentInstance.resizeArea.nativeElement.dispatchEvent(dblclick);
```

Comment:
Please check if `dblclick` event could be triggered through `UIInteractions.simulateMouseEvent`.